camel-core/src/main/java/org/apache/camel/component/seda/SedaComponent.java

```diff
@@ -35,12 +36,12 @@
  */
 public class SedaComponent extends UriEndpointComponent {
     protected final Logger log = LoggerFactory.getLogger(getClass());
-    protected final int maxConcurrentConsumers = 500;
+    protected final int maxConcurrentConsumers = SedaConstants.MAX_CONCURRENT_CONSUMERS;
 
-    @Metadata(label = "consumer", defaultValue = "1")
-    protected int concurrentConsumers = 1;
-    @Metadata(label = "advanced")
-    protected int queueSize;
+    @Metadata(label = "consumer", defaultValue = ""+SedaConstants.CONCURRENT_CONSUMERS)
```

Comment:
Small format issues on this and next lines, space is missing

Reply:
I am not sure what space is missing . I have run mvn compile -P sourcecheck and passed all checks

Reply:
You must see errors like ''+' is not preceded with whitespace.' but build doen't stop as expected.
Please run and check output again.
@oscerd do you have any ideas why maven-checkstyle-plugin:3.0.0:checkstyle become skips build on audit errors?

Reply:
It's been a while since we remove this feature, if we notice problem with CS we can fix them later.

Reply:
@oscerd thanks, missed this changes.

Reply:
I noticed that days ago, you're not alone :-)